database/factories/GenreFactory.php

```diff
@@ -21,8 +21,9 @@ class GenreFactory extends Factory
      */
     public function definition()
     {
+
         return [
-            //
+
```

Comment:
I suggest you have a default value for this.

`return [
      'name' => Genre::GENRE_ACTION
]` 

Just so you can use it for testing ;D

Reply:
I have added only a single entry here Vick, just like your example above, is that right? hehe